manifests/config.pp

```diff
@@ -61,5 +61,12 @@
     group   => 'root',
     content => $::openvpn::dh,
   }
+  file { '/etc/openvpn/keys/ta.key':
+    ensure  => present,
+    mode    => '0600',
+    owner   => 'root',
+    group   => 'root',
+    content => $::openvpn::ta,
```

Comment:
If `$ta` remains `undef`, will this cause side-effects? Does Puppet create a 0-byte file, or does the manifest break?